src/Server/TCPHandler.cpp

```diff
@@ -1313,9 +1319,11 @@ void TCPHandler::receiveQuery()
             session->authenticate(AlwaysAllowCredentials{client_info.initial_user}, client_info.initial_address);
         }
 #else
-        throw Exception(
+        auto exception = Exception(
             "Inter-server secret support is disabled, because ClickHouse was built without SSL library",
-            ErrorCodes::SUPPORT_IS_DISABLED);
+            ErrorCodes::AUTHENTICATION_FAILED);
+        session->onAuthenticationFailure(AlwaysAllowCredentials{USER_INTERSERVER_MARKER}, socket().peerAddress(), exception);
```

Comment:
`USER_INTERSERVER_MARKER` is only a marker in TCP procotol, it's not the name of some user.
This marker was invented only because we had to add fields to the hello package.
If possible let's avoid writing this marker to any kind of logs as a user name.